NBitcoin/UnsignedList.cs

```diff
@@ -20,7 +20,7 @@ public UnsignedList(Transaction parent)
 		public Transaction Transaction
 		{
 			get;
-			internal set;
+			set;
 		}
```

Comment:
why?

Reply:
Yes, I tried reverting this back to internal set and my code still builds. Must have been for an iteration that I didn't end up needing. I will revert this back to internal.